frontend/lib/error/index.ts

```diff
@@ -19,6 +19,7 @@ interface PACTAError extends NuxtError {
 }
 
 export const createErrorWithRemediation = (err: string | Partial<NuxtError>, r: Remediation): PACTAError => {
+  console.log(err)
```

Comment:
remove? This should get logged up the stack if it's thrown correctly

Reply:
These aren't showing up in logs, and it's proving challenging to debug stack traces without them.